packages/astro/src/runtime/client/dev-overlay/plugins/audit/a11y.ts

```diff
@@ -352,13 +352,72 @@ export const a11y: AuditRuleWithSelector[] = [
 	},
 	{
 		code: 'a11y-missing-content',
-		title: 'Missing content on element important for accessibility',
-		message: 'Headings and anchors must have content to be accessible.',
+		title: 'Missing content',
+		message:
+			'Headings and anchors must have an accessible name, which can come from: inner text, aria-label, aria-labelledby, an img with alt property, or an svg with a tag <title></title>.',
```

Comment:
Just checking because I don't think these error messages currently get pulled into docs, so this should be fine. But for docs, we would be using back ticks for inline code formatting on all values, but especially any components like `<title>` or else docs will break.

@Princesseuh - tangentially, do we *want* these dev tool error messages available in docs somewhere? Eventually? We don't really have any "using dev toolbar" docs at all. Is this something we want/care about?

Reply:
Sorry, missed this message (blame GitHub notifications not working properly since a few weeks 😠). Yes, eventually I would like to have the audits in the docs, similar to the error reference.